src/CppUTestExt/IEEE754ExceptionsPlugin.cpp

```diff
@@ -71,7 +71,7 @@ void IEEE754ExceptionsPlugin::enableInexact()
 
 bool IEEE754ExceptionsPlugin::checkIeee754ExeptionFlag(int flag)
 {
-    return fetestexcept(flag);
+    return !! fetestexcept(flag);
```

Comment:
Was going to make PR but my git global setup is messed up and commit CRLF to the repo :-(. Had to play around with Cygwin and it is all messed up, bash, git and all. Trying to fix asap....